docs/user-guide/kubernetes.md

```diff
@@ -330,6 +330,72 @@ echo "$(minikube ip) traefik-ui.minikube" | sudo tee -a /etc/hosts
 
 We should now be able to visit [traefik-ui.minikube](http://traefik-ui.minikube) in the browser and view the Træfik Web UI.
 
+## Basic Authentication
+
+It's possible to add additional authentication annotations in the Ingress rule.
+The source of the authentication is a secret that contains usernames and passwords inside the key auth.
+To read about basic auth limitations see the [Kubernetes Ingress](/configuration/backends/kubernetes) configuration page.
+
+#### Creating the Secret
+
+A. Use `htpasswd` to create a file containing the username and the base64-encoded password:
+
+```shell
+htpasswd -c ./auth myusername
+```
+
+You will be prompted for a password which you will have to enter twice.
+`htpasswd` will create a file with the following:
+
+```shell
+cat auth
+```
+```
```

Comment:
Are the extra triple backticks intentional? It looks a bit strange on the rendered output.

I'd suggest to simply drop the `cat auth` part. It's clear from the paragraph right above that we're talking about the content of the `auth` file, no need to show the extra `cat` call IMHO.

Reply:
I have split the block: because it's more easy to copy-paste (with the end of block button)

Reply:
OK :+1: